core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/DetailedSampleBuilder.java

```diff
@@ -43,8 +43,11 @@
 import uk.ac.bbsrc.tgac.miso.core.exception.ReportingException;
 import uk.ac.bbsrc.tgac.miso.core.security.SecurableByProfile;
 
-public class DetailedSampleBuilder implements DetailedSample, SampleAliquot, SampleStock, SampleTissue, SampleTissueProcessing,
-    SampleCVSlide, SampleLCMTube, Identity {
+public class DetailedSampleBuilder
+    implements DetailedSample, SampleAliquot, SampleStock, SampleTissue, SampleTissueProcessing, SampleCVSlide, SampleLCMTube, Identity {
+  public interface InferStockFromAliquot {
```

Comment:
From a quick skim, this person doesn't seem to understand what verbs and nouns even are...